primefaces/src/main/java/org/primefaces/util/FileUploadUtils.java

```diff
@@ -314,6 +331,15 @@ private static boolean isValidFileContent(PrimeApplicationContext context, FileU
         return true;
     }
 
+    private static String getFileExtension(String fileName) {
```

Comment:
You can use `org.apache.commons.io.FilenameUtils#getExtension`

Reply:
done